team_meetings.yml

```diff
@@ -281,6 +281,11 @@ events:
       interval:
         weeks: 2
       until: 2022-12-31
+      except_on:
+        - 2022-11-17 14:05:00
+      #also_on:
+        # - 2022-11-16 11:05:00
```

Comment:
Imho this should be removed. but no blocker.

Reply:
+1. But that would be a nice addition to https://github.com/scientific-python/yaml2ics

Reply:
I was wondering whether such a feature exists, searched the yaml2ics docs (which consists of a few examples only AFAICS) and then commented this out, leaving it there for human beings. I'll drop it.
If I find the time, I'll open a feature request for yaml2ics.